CI/Dockerfile

```diff
@@ -179,9 +179,6 @@ RUN mkdir -p ${moab_build_dir}/build && \
                   -DBUILD_SHARED_LIBS=ON \
                   -DENABLE_FORTRAN=OFF \
                   -DENABLE_PYMOAB=ON && \
-    mkdir -p ${moab_build_dir}/build/pymoab/lib/python3.6/site-packages && \
-    PYTHONPATH=${moab_build_dir}/build/pymoab/lib/python3.6/site-packages:${PYTHONPATH} make -j${ci_jobs} && \
-    PYTHONPATH=${moab_build_dir}/build/pymoab/lib/python3.6/site-packages:${PYTHONPATH} make install && \
```

Comment:
This took out too many lines!  Need to keep the `make` lines:
```
make -j${ci_jobs} && \
make install && \
```

Reply:
just pushed a fix, CI is [rerunning](https://github.com/shimwell/DAGMC/actions/runs/4823792354)

Reply:
CI is passing again :tada:

Reply:
also it ran in just 23 mins. Nice and quick